app/javascript/components/ResultsPanel.js

```diff
@@ -1,117 +1,60 @@
-  },
-  {
-    type:'study',
-    name: 'Single nucleus RNA-seq',
-    cell_count: 5426,
-    acession: 'SCP4',
-    description: 'orem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
-  },
-  {type:'study',
-    name: 'Single nucleus RNA-seq',
-    cell_count: 5426,
-    acession: 'SCP5',
-    description: 'orem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
-  },
-];
-
-class ResultsPanel extends React.Component{
-  constructor(props){
-    super(props);
-    this.state = {
-      results: undefined,
-      facets : []
-    };
-    this.handleStudyLabel = this.handleStudyLabel.bind(this);
-    };
-
-    handleStudyLabel(studyURL){
-      // This eventually will be an API call that will render the associated
-      // Study page 
-      alert({studyURL})  
-    }
+const ResultsPanel = (props) => {
     
-    showResults(){
-      // Function to hande future pagination
-    }
-    render(){
-      
-      return(
-        <div>
-          <Tabs defaultActiveKey='study' transition={false}>
+  return( 
+    <div id="results-panel">
+      <Tab.Container id="result-tabs" defaultActiveKey="study">
+          <Tabs defaultActiveKey='study' animation={false} >
             <Tab eventKey='study' title="Studies" >
-              <StudyResults handleStudyLabel = {this.handleStudyLabel} results={this.state.results}/>
+              <StudyResults results={props.results}/>
             </Tab>   
-            <Tab eventKey='files' title='Files'/>       
-        </Tabs>
-        </div>
+            <Tab eventKey='files' title='Files'/>   
+          </Tabs>
+      </Tab.Container>
+    </div>
   );
-  
-    }
+
   }
 
 const StudyResults = (props) => {
-  // This component may have state/props where onMount a API call is made
-  // to get the initial results on the home page.
+  var results;
```

Comment:
I like how you structured this to handle empty results.  let's just rename this to 'displayedResults' so it's not as easily confused with props.results

Reply:
5160d89